lib/data_update_scripts/20210503175027_move_summary_and_location_to_profile.rb

```diff
@@ -0,0 +1,29 @@
+module DataUpdateScripts
+  class MoveSummaryAndLocationToProfile
+    def run
+      # do we also need to delete the old data?
+      ActiveRecord::Base.connection.execute(
+        <<~SQL.squish,
+          BEGIN;
```

Comment:
you should wrap `ActiveRecord::Base.connection.execute` with `ActiveRecord::Base.transaction` instead of using this. Tests will correctly use `SAVEPOINT` instead of `BEGIN transaction;` since they run in a transaction that's rolled back and Rails will do the right thing in production as well, using `BEGIN`